presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -5959,6 +5959,38 @@ public void testTry()
     }
 
     @Test
+    public void testLambdaFunctionWithTry()
+    {
+        assertQuery("SELECT TRY(apply(5, x -> x + 1) / 0)", "SELECT NULL");
+        assertQuery("SELECT TRY(apply(5 + RANDOM(1), x -> x + 1) / 0)", "SELECT NULL");
+        assertQueryFails("SELECT apply(5 + RANDOM(1), x -> x + TRY(1 / 0))", "Try expression inside lambda expression is not support yet");
+    }
+
+    @Test
+    public void testTryInvalidJsonParse()
+    {
+        assertQuery("SELECT TRY (JSON 'INVALID')", "SELECT NULL");
+        assertQuery("SELECT TRY (JSON_PARSE('INVALID'))", "SELECT NULL");
+    }
+
+    @Test
+    public void TestTryExpression()
```

Comment:
Merge them into `testTry`, do the same for `testTryInvalidJsonParse`

Reply:
You forgot to merge these tests into `testTry` :). 

After merge, add a comments stating these tests doesn't have FROM clauses and might be constant folded.